files/en-us/learn/html/tables/structuring_planet_data/index.html

```diff
@@ -59,7 +59,7 @@ <h2 id="Steps_to_complete">Steps to complete</h2>
  <li>Create all the content rows inside the table body, remembering to make all the row headings into headings semantically.</li>
  <li>Ensure all the content is placed into the right cells — in the raw data, each row of planet data is shown next to its associated planet.</li>
  <li>Add attributes to make the row and column headers unambiguously associated with the rows, columns, or rowgroups that they act as headings for.</li>
- <li>Add a black border just around the column that contains all the planet name row headers.</li>
+ <li>Add a black <a href='https://developer.mozilla.org/en-US/docs/Web/CSS/border'>border</a> just around the column that contains all the planet name row headers.</li>
```

Comment:
Please use an absolute path, so that it works on intermediate deployments, too:
```suggestion
 <li>Add a black <a href='/en-US/docs/Web/CSS/border'>border</a> just around the column that contains all the planet name row headers.</li>
```

Reply:
@Ryuno-Ki I have the suggested changes and out of curiosity just want to how 
`'https://developer.mozilla.org/en-US/docs/Web/CSS/border` is different from `/en-US/docs/Web/CSS/border`

I tested both of these options and they work fine .

Reply:
Yeah both will work, but the policy is to always use just the `/en-US/docs/Web/CSS/border` path rather than the full `https://developer.mozilla.org/en-US/docs/Web/CSS/border` URL. And the flaw checker in Yari enforces that; that is, Yari consider any full `https://developer.mozilla.org/…` URL in an article to be a flaw that needs to be fixed (and which is a “fixable flaw” hat Yari can fix automatically).